### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by prateksha**
Introducing Team Rubies 2018!

### CODE REVIEW

File: blog/_posts/2018-07-09-IntroducingTeamRubies.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-IntroducingTeamRubies.md]: Hi @prateksha ! Thanks for the PR. Extra one line space at the start is unnecessary. Please remove it.

File: blog/_posts/2018-07-09-IntroducingTeamRubies.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+---
+title: Introducing Team Rubies:tada:
+layout: post
+created_at: Mon July-09-2018
+permalink: blog/2018-07-09-introducing-team-rubies
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-IntroducingTeamRubies.md]: Permalink should be same as file name [it is case-sensitive]. :)

File: blog/_posts/2018-07-09-IntroducingTeamRubies.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+**Raounaq** - Raounaq is a passionate software developer who uses Ruby to express in code what defines him. Apart from coding he played for a football club from Delhi, India. He also likes to travel.
+
+**Brihi** - Atibhi's childhood friend, RGSoC 2017 scholar and a person who loves Code, Coffee and Music. Brihi can work with anything that is associated with Data, Machine Learning, and AI. Even though she isn't their official coach, RGSoC 2018 wouldn't have been possible without her. From reviewing their application to jumping with joy when they got selected, she has been their biggest supporter throughout the process !
+
+### **Choosing a project and interacting with the mentors !**
+
+They had chosen their project even before the list of official projects had been announced. They were thrilled when their primary choice "if-me" got selected for RGSoC'18.
+They could relate to "if-me" and wanted to contribute to the mental health community. Here is their first contact with Julia, their mentor way before the official list was announced.
+
+![FirstContact](/img/blog/2018/2018-07-09-TeamRubies-FirstContact.png)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-IntroducingTeamRubies.md]: Please add pic credits like other images for this photo/screenshot [in the text].

File: blog/_posts/2018-07-09-IntroducingTeamRubies.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+On 5th April, they got a mail for the first round of interview.This was their first ever interview and they had butterflies in their stomach!
+After the interview, they had their fingers crossed and were eagerly waiting for the results.
+
+Soon it was the time for results! 17th April was finally here! Both of them were anxious and were continuously checking their mails. At 12:30 AM they received a mail saying that there was another round of interview. :sweat_smile:
+
+They thought that their first interview wasn't good enough and this was the final chance for them to redeem themselves.
+By 19th April they had gone through their proposal multiple times and brushed up their technical knowledge. The interview was scheduled to start at 9 PM and they sat nervously in front of their laptops at 8:30 PM. At 9 PM the interview started. Little did they know that this is what RGSoC meant by "interview".
+
+![HappyUs](/img/blog/2018/2018-07-09-TeamRubies-HappyUs.jpeg)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-IntroducingTeamRubies.md]: Pic description and pic credits missing. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by prateksha** [comment]
@nerdyvaishali Made all the required changes!

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**